python/lib/sift_py/data_import/csv.py

```diff
@@ -181,12 +181,12 @@ def is_uint64(n: pd.Series) -> bool:
         data_config = {}
         df = pd.read_csv(path, skiprows=skip_rows)
 
-        units: Optional[List[str]] = None
+        units: List[str] = []
         if units_row is not None:
             df_units = pd.read_csv(path, nrows=units_row)
             units = cast(List[str], df_units.iloc[units_row - 1].astype(str))
 
-        descriptions: Optional[List[str]] = None
+        descriptions: List[str] = []
         if descriptions_row is not None:
             df_descriptions = pd.read_csv(path, nrows=descriptions_row)
             descriptions = cast(List[str], df_descriptions.iloc[descriptions_row - 1].astype(str))
```

Comment:
dont think we need these casts anymore either

Reply:
Mypy complains because astype is a `Series[bool]`.